src/applications/facility-locator/components/VABenefitsCall.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { parsePhoneNumber } from '../utils/phoneNumbers';
+import Telephone from '@department-of-veterans-affairs/formation-react/Telephone';
+
+const { formattedPhoneNumber, extension, contact } = parsePhoneNumber(
+  '800-827-1000',
+);
+
+export default function VABenefitsCall() {
+  return (
+    <div>
+      <p className="p1">
+        <strong>To check on the status of your claim or appeal</strong>, use our{' '}
+        <a href="/claim-or-appeal-status">online claim status tool</a>.
+      </p>
+      <p className="p1">
+        <strong>To get help with benefits</strong>, call{' '}
+        <Telephone contact={contact} extension={extension}>
+          {' '}
+          {formattedPhoneNumber}
+        </Telephone>{' '}
```

Comment:
You can just use the constant here:

```js
        <Telephone contact={CONTACTS.VA_BENEFITS} />
```

Reply:
Tried this but fails 
<img width="1422" alt="Screen Shot 2020-07-15 at 4 31 12 PM" src="https://user-images.githubusercontent.com/100468/87598173-a7387a80-c6b8-11ea-85bd-5aee54438841.png">

Reply:
never-mind, that was my bad, I was using the import incorrectly